packages/lit-dev-content/site/tutorials/content/svg-templates/01.md

```diff
@@ -0,0 +1,169 @@
+This section of the tutorial covers the basics of using SVG with Lit
+including how to:
+
+- Create SVG templates
+- Compose HTML templates from SVG templates
+- Apply reactive-properties to SVG templates
+- Encapsulate SVG inside the shadow DOM
+
+### Learn
+
+In [pattern making](https://www.laurenceking.com/products/how-to-make-repeat-patterns),
+an _element_ refers to the smallest piece of a
+pattern that cannot be derived from other parts of the pattern.
+
+The _element_ in this demo will be a single piece of text. We can
+create text in SVG and Lit with _SVG templates_ as demonstrated below.
+
+```ts
+const helloSVG = svg`<text>Hello, SVG!</text>`;
+```
+
+In `Lit`, an SVG template must be the child of an `<svg>` element. The
+`<svg>` element must be contained inside an [HTML template](https://lit.dev/docs/templates/overview/).
+An implementation of these requirements is demonstrated by the following
+example.
+
+```ts
+const helloHTML = html`
+  <svg>
+    ${svg`<text>Hello, SVG!</text>`}
+  </svg>
+`;
+```
+
+### Apply
+
+Create a function called `createElement` that generates a `<text>` 
+element with configurable display text using a Lit SVG template.
+
+{% switchable-sample %}
+
+```ts
+const createElement = (chars: string): SVGTemplateResult => svg`
+  <text>${chars}</text>
+`;
+```
+
+```js
+import {LitElement, html, svg} from 'lit';
+
+const createElement = (chars) => svg`
+  <text>${chars}</text>
+`;
+```
+
+{% endswitchable-sample %}
+
+Add text-specfic [styling attributes](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/text#global_attributes)
```

Comment:
Revisiting the function we just made is annoying. Make one step? (defer to @e111077's opinion).

Reply:
i merged the two steps and the instructions
sounds good to me